conbench/templates/index.html

```diff
@@ -28,7 +28,7 @@ <h4>Recent runs for {{reponame}}</h4>
     </tr>
     <tr>
         <!-- add tooltip, saying that this is DB insertion time -->
-        <th scope="col" style="width: 180px">insertion time</th>
+        <th scope="col" style="width: 180px">insertion time (result count)</th>
```

Comment:
What do you think about adding this as a separate column? I know we want to generally keep those to a minimum, but if it's a separate column it becomes filter-able and sort-able. IMO if we're going to expose this kind of info, we should be able to filter | sort by it.

Reply:
> What do you think about adding this as a separate column? I know we want to generally keep those to a minimum, but if it's a separate column it becomes filter-able and sort-able. IMO if we're going to expose this kind of info, we should be able to filter | sort by it.

I was hoping for that feedback :rocket:  I agree, my favorite choice was to add a new column, but I thought I keep it more minimal first and simply ask for feedback.